src/content/reference/manifest.mdx

```diff
@@ -610,6 +610,29 @@ volumes:
 
 Sets the working directory of your development container.
 
+### forward ([string], optional)
+
+You can also declare global forwards that will be used by all dev environments.
+
+As in `forward` in dev section, several formats are accepted for this, but it is necessary to indicate the name of the service to be forwarded using the notation `localPort:remoteService:remotePort`.
+
+```yaml
+forward:
+  - 5432:postgres:5432
+  - localPort: 27017
+    remotePort: 27017
+    name: mongodb
+```
+
+The global forwards can be used by each declared development environment and can be used at the same time without any port collisions. 
```

Comment:
I would skip the rest and just say that okteto handes these forwards to avoid port collisions if two `okteto up` sessions are running at the same time